src/controllers/factorycontroller.js

```diff
@@ -105,6 +119,22 @@ class FactoryController extends OrderController
         return true;
     }
 
+    extraTruck (formValues)
+    {
+        if (this.validateTruck(truck)) {
+            GAME.model.config.money - GAME.model.config.costExtraTruck);
+            GAME.model.config.maxSimultaneousOrders +1;
```

Comment:
Same thing here, this update needs to be assigned (you may use `GAME.model.config.maxSimultaneousOrders++` too, that re-assigns in-place)